.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: node_js
-
 node_js:
   - 0.10
-  - 0.11
```

Comment:
how about instead of this, we add iojs (assuming we arent using mock-fs anywhere, i cant recall) and maybe 0.12. It doesnt seem valuable, but making sure the build works on mainstream node releases is important to me